shared/css/color.scss

```diff
@@ -1,9 +1,11 @@
 // colors from https://www.dropbox.com/s/ycjiwt8vxlmv2ih/Code.org_stylesheet_v2_09.05.13.pdf
 
+$black: #000000;
 $charcoal: #5b6770;
 $light_gray: #949ca2;
 $lighter_gray: #c6cacd;
 $lightest_gray: #e7e8ea;
+$white: #ffffff;
```

Comment:
True; maybe we should do this only for the [17 CSS2 colors](http://stackoverflow.com/a/8561913)?

I could go either way.